hnn_core/network_builder.py

```diff
@@ -490,8 +492,8 @@ def _record_spikes(self):
     def aggregate_data(self):
         """Aggregate somatic currents, voltages, and dipoles."""
         for cell in self.cells:
-            if cell.celltype in ('L5_pyramidal', 'L2_pyramidal'):
-                self.dipoles[cell.celltype].add(cell.dipole)
+            if cell.name in ('L5Pyr', 'L2Pyr'):
+                self.dipoles[_long_name(cell.name)].add(cell.dipole)
```

Comment:
Is it possible for us to finally consolidate the short and long name usage?

Reply:
I think we certainly can and should. We just need to get to it. I would leave it for another PR though ...

Reply:
Probably a good starting point for one of the undergrads